### PR TITLE
Propose new OSRAM device

### PR DESCRIPTION
**Pull request by petrpulc**

### CODE REVIEW

File: src/devices/osram.ts [hunk 1]
```diff
@@ -254,6 +254,13 @@ const definitions: Definition[] = [
         description: 'SMART+ Candle E14 Dimmable White',
         extend: [ledvanceLight({})],
     },
+    {
+        zigbeeModel: ['Control box TW'],
+        model: 'Control box TW',
```
Comment (Koenkk) [inline src/devices/osram.ts:259]: Could you use the model number that is written on the device here?

Comment (Koenkk) [reply src/devices/osram.ts:259]: Is there some EAN? E.g. on the site where you bought it.

Comment (petrpulc) [reply src/devices/osram.ts]: Finally was able to get the controller out and copy the model number. Please @Koenkk consider a re-review.

**Comment by Koenkk** [comment]
Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `AB390020055.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Pull request merged**